tests/Browser/smoke.testcafe

```diff
@@ -1,2053 +0,0 @@
-{
```

Comment:
I believe this file is used by Testcafe Studio™ to allow editing the fixtures through their UI. If we're okay not using that product again I'm fine with deleting this.

Reply:
I didn't, and hadn't, and wasn't going to probably.